src/router/index.ts

```diff
@@ -2,14 +2,29 @@ import { createRouter, createWebHistory } from "vue-router";
 
 const routes = [
   {
-    path: "/",
+    path: "/:quizId",
     name: "Player",
+    props: true,
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
```

Comment:
let's remove this no
we can just put a reference link instead of explaining https://router.vuejs.org/guide/advanced/lazy-loading.html